src/nft/components/details/detailsV2/TraitRow.tsx

```diff
@@ -27,21 +45,32 @@ const TraitRowValue = styled(ThemedText.BodySmall)<{ $flex?: number; $justifyCon
   justify-content: ${({ $justifyContent }) => $justifyContent};
 `
 
-export const TraitRow = ({ trait }: { trait: Trait }) => {
+export const TraitRow = ({ trait, collectionAddress }: { trait: Trait; collectionAddress: string }) => {
+  const [rowHovered, toggleRowHovered] = useReducer((s) => !s, false)
```

Comment:
nit: could you use CSS instead of React? For example, always render the trait caption (e.g. `very rare`), but let it have `opacity: 0` or `display: none` and change when parent hovers?

Reply:
I like this solution, went with opacity